csatar-plugins/forms/models/Form.php

```diff
@@ -29,9 +30,8 @@ public function beforeSave()
     {
         $model = '\\' . $this->model;
         if (! class_exists($model)) {
-            $error = "Error: The form's model could not be found.\n"."\t Please make sure you enter correct path and model name.";
+            $error = e(trans('csatar.forms::lang.errors.formModelNotFound'));
             throw new ApplicationException($error);
-//            throw new ValidationException(['model' => $error ]);
         }
```

Comment:
1. Mi van ha valaki egyenesen úgy írja be a model nevet, hogy már van az elejént `\`?
2. Megintcsak: az `ApplicationException` túl általános meg kell nézni, hogy nincs-e már valami előre definiált Exception, amit lehetne használni, vagy csinálni egy saját exception-t

Reply:
1. Hibaüzenet kap, hogy javítsa ki.
2. Átírtam ValidationException-re.